client/tests/integration_test.rs

```diff
@@ -747,6 +774,48 @@ mod put_deploy {
     }
 }
 
+mod rate_limit {
+    use super::*;
+    use casper_node::types::Timestamp;
+
+    #[tokio::test(threaded_scheduler)]
+    async fn client_should_should_be_rate_limited_to_approx_1_qps() {
+        // Transfer uses PutDeployParams + PutDeploy
+        let server_handle = Arc::new(MockServerHandle::spawn::<PutDeployParams>(
+            PutDeploy::METHOD,
+        ));
+
+        let now = Timestamp::now();
+        // Our default is 100 req/min, so this will time out
+        for i in 0..10u32 {
+            println!("request {}", i);
+            let amount = "100";
+            let maybe_target_account =
+                "01522ef6c89038019cb7af05c340623804392dd2bb1f4dab5e4a9c3ab752fc0179";
+
+            // block_in_place is needed to prevent deadlock on the futures::executor::block_on
+            // call inside transfer
+            let server_handle = server_handle.clone();
+            let join_handle = tokio::task::spawn_blocking(move || {
```

Comment:
This has been missed too.

Reply:
I agree I completely missed this, but I wanted to say, I don't personally find much value in this kind of change, in fact it hides meaning. It makes `tokio::task` -> `task` and there are other libraries that implement the same meaning of `task`, like `async_std`. Same applies for `futures::executor` back then `tokio::executor` was still a thing. I'm happy with adjusting to your preference on this case, but I'd like to better understand your reasoning for it.

Reply:
It's purely due to that being the code style since I started here.  My own preference is almost always for consistency, so since we have this style elsewhere, I prefer to see it applied everywhere.

If you fancy changing the style guide, we should probably discuss outside of a PR (I think you'd likely get some support for your POV).